README.md

```diff
@@ -19,3 +20,18 @@ to be the most efficient one!
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## My Runtime Analysis
+The big $\Theta$ complexity of my implementation is $v^3+ev^2$. This is because 
+when we start we have to go through every node to initialize all the nodes to 
+infinity, then we have to set the initial node to 0. This takes v time to do.
+Next, we have to go through a foe loop which will run v times as well, since we
+have to go through all the nodes at least once taking v time. After that, I 
+reverse sort the list so that the next node to be checked is at the top of a queue
+which takes $v^2$ time. Next, we have to go through all the edges in the graph,
```

Comment:
The sorting would be more efficient and easier if you use the built-in sorting function.